locale/nl/LC_MESSAGES/clock-override.po

```diff
@@ -0,0 +1,68 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-08-15 18:48+0200\n"
+"PO-Revision-Date: 2020-11-02 13:48+0100\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: Heimen Stoffels <vistausss@outlook.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: nl\n"
+
+#: prefs.js:51
+msgid "Text to display instead of the clock"
+msgstr "De in plaats van de klok te tonen tekst"
```

Comment:
This one is a bit hard to parse I think. Maybe something like:

```suggestion
msgstr "De te tonen tekst in plaats van de klok"
```

Or:

```suggestion
msgstr "De tekst op de plek van de klok"
```

Reply:
I've never heard any user complaints about such sentences, but I agree that "De tekst op de plek van de klok" is better. Thanks for your collaboration! :)